src/MediaBundleForm.php

```diff
@@ -209,6 +215,7 @@ protected function copyFormValuesToEntity(EntityInterface $entity, array $form,
   public function save(array $form, FormStateInterface $form_state) {
     /** @var  \Drupal\media_entity\MediaBundleInterface $bundle */
     $bundle = $this->entity;
+    $bundle->setNewRevision($form_state->getValue('new_revision'));
```

Comment:
I tested this manually and it doesn't seem to be. Steps:
- create media bundle (check "Create new revision")
- create new media entity of the bundle I just created ("Create new revision" checkbox is checked as new entity is being created)
- Navigate back to edit page for the entity I just created
- **Expected result:** "Create new revision" is checked by default
- **Actual result:** Checkbox is not checked

Am I missing something?